.changes/next-release/feature-pythonsupport-57070.json

```
@@ -0,0 +1,5 @@
+{
+  "type": "feature",
+  "category": "python support",
+  "description": "Support python 3.11 for v1 of the CLI."
```

We typically start with verbs like "Add", "Update", "Fix", "Remove". Something like this may be more consistent with existing verbiage.
```suggestion
  "description": "Add support for Python 3.11"
```